tests/README.md

```diff
@@ -0,0 +1,81 @@
+## Backend development
+
+The backend test suite can be run locally. 
+
+### Start the services
+
+We need all but the main `backend` service running.  Since multiple `backend` instances are allowed, we will simply start everything up
+
+    # ENSURE THE IMAGES ARE CLEAN
+    docker-compose down
+    # SETUP ALL IMAGES
+    docker-compose up --build
+
+### Install into virtual environment
+
+Treeherder, requires specific library versions that will likely interfere with with what you have installed. It is best to setup a virtual environment to contain the quirks it introduces.
+
+Be sure you are in the `treeherder` main directory
+
+    pip install virtualenv
+    virtualenv venv
```

Comment:
I have not tested `python3 -m venv venv` , let's leave it as is with plans to use poetry

Reply:
...fixed anyway

Reply:
...nope the builtin venv results in hash conflicts and nasty file locks on windows.  Probably not stable yet.

Reply:
If that is the case please file it upstream with Python because that feature has been around since 3.4.

Could you also post the issue you're facing?

Reply:
> Could you also post the issue you're facing?

No, I will make an issue.  Exploring why commands do not work, when we have working commands, is outside the scope of the PR. 

https://bugzilla.mozilla.org/show_bug.cgi?id=1613127